### PR TITLE
Combine tests for checking for a navigable toolbar into one test to improve performance

### PR DESCRIPTION
**Pull request by jeryj**
## What?

Taking several related tests and combining into one test to save time on running tests.

This needs to wait to be merged until https://github.com/WordPress/gutenberg/pull/51548 lands, as combining these tests uncovered a real bug in the keyboard navigation behavior.

## Testing Instructions

`npm run test:e2e:playwright test/e2e/specs/editor/various/toolbar-roving-tabindex.spec.js`

### CODE REVIEW

**Review by kevin940726 — APPROVED** [review]
LGTM 👍 💯

**Review by Mamaduka — APPROVED** [review]
Thank you, @jeryj!

**Comment by Mamaduka** [comment]
@scruffian, the PR description has a note to hold off merging this until the #51548 is merged.

**Comment by scruffian** [comment]
Oh sorry :(

@jery should we merge #51548, or revert this?

**Comment by jeryj** [comment]
It merged into `fix/shift-tab-form-elements` so it's OK! It just means the same code will get reviewed again on that branch, which is no problem. It would only have been an issue if it got merged to `trunk`, in which case the tests would start breaking.

**Pull request merged**
